README.rst

```diff
@@ -149,31 +143,35 @@ Testing
 
 To run the Rubicon test suite:
 
-1. Configure your shell environment to point to a ``PYTHON_CONFIG`` binary.
-   `virtualenv` and `venv` do not necessarily put the ``python3-config`` binary
-   on your ``$PATH``, but you can use this expression to set it properly::
+1. Check if `python-config` passes exists in your environment, and points at
```

Comment:
> Check if `python-config` passes exists

I think you mean just "exists" not "passes exists".

Reply:
I also think we should standardize on `python3-config`, both here in the docs, and the `Makefile`. Not a showstopper.

Reply:
Fair enough. Changed.